apps/client/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,143 @@
+import NextAuth from 'next-auth'
```

Comment:
I'm personally weary about leaning into Next's backend functionality. The future market for Maybe applications is likely many self-hosted instances, and serverless is a better fit for applications with variable load traffic.

I think it's fine to merge this for now, but honestly I'd like to replace the entire backend with a Fastify + NestJS server.

Reply:
+1 for NestJS. We leverage it at work and while it can be a bit verbose at times, it's great once you learn it.

Reply:
+1 for this. Can help with migrating it to NestJS as well